cli/tests/integrations/test_package.py

```diff
@@ -671,6 +669,46 @@ def test_search_table():
     assert stderr == b''
 
 
+def test_search_ends_with_wildcard():
```

Comment:
the author mistakenly added them to the end of the file, which should be where private functions go.

I often find minor issues like this when on separate tasks.  it's too much overhead to split them up into separate branches.